extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -56,6 +60,10 @@ require([
 
 	// Everything starts after content and JS
 	win.wgAfterContentAndJS.push(function () {
+		if (floatingRail) {
+			pageLevelParams.add('rrspace', floatingRail.getAvailableSpaceParameter().toString());
```

Comment:
I couldn't set it in adPageLevelParams module, because this modules is used by AnalyticsProviders and it would required adding adHelper and jQuery (!) to the bidders package (which should be as light as possible). I think we can do something about that.